Plugins/Sycl/src/Seeding/CreateSeedsForGroupSycl.cpp

```diff
@@ -219,20 +277,88 @@ void createSeedsForGroupSycl(
     }  // sync
 
     if (edgesBottom > 0 && edgesTop > 0) {
-      // Global and local range of execution for edgesBottom number of threads.
-      // Local range corresponds to block size.
+      // Global and local range of execution for edgesBottom number of
+      // threads. Local range corresponds to block size.
       cl::sycl::nd_range<1> edgesBotNdRange{
-          cl::sycl::range<1>(numWorkItems(edgesBottom, maxWorkGroupSize)),
+          cl::sycl::range<1>(num1DWorkItems(edgesBottom, maxWorkGroupSize)),
           cl::sycl::range<1>(maxWorkGroupSize)};
 
       // Global and local range of execution for edgesTop number of threads.
       cl::sycl::nd_range<1> edgesTopNdRange{
-          cl::sycl::range<1>(numWorkItems(edgesTop, maxWorkGroupSize)),
+          cl::sycl::range<1>(num1DWorkItems(edgesTop, maxWorkGroupSize)),
           cl::sycl::range<1>(maxWorkGroupSize)};
 
-      // We store the indices of the BOTTOM/TOP space points of the edges of the
-      // bottom-middle and top-middle bipartite duplet graphs.
-      // They index the bottomSPs and topSPs vectors.
+      // EXPLANATION OF INDEXING (fisrt part)
+      /*
+        (for bottom-middle duplets, but it is the same for middle-tops)
```

Comment:
You could consider also adding these comments to the rST document.

Reply:
I'm not sure about that. :/ I looked at other rST documents, and found that they mostly talk about well physics or things concerning users, how to build or use a specific class. Indexing and memory management is a bit different from that. But if you think it would be nice to add them there too, I can do that.